src/main/java/core/shop/service/impl/FileServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.shop.service.impl;
+
+import core.shop.service.FileService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileServiceImpl implements FileService {
```

Comment:
It needs to be divided into two interfaces.
A separate class with reading from a file, a separate class with writing to a file.